packages/integration-test/tests/cases/context.ts

```diff
@@ -63,15 +63,16 @@ export async function terminate(ctx: CtxProvider) {
     wallets,
     provider,
   } = ctx()
-  provider.disconnect(0, 'exit')
   await Promise.all([
     coordinator.stop(),
     wallets.alice.node.stop(),
     wallets.bob.node.stop(),
     wallets.carl.node.stop(),
     wallets.coordinator.node.stop(),
   ])
-  await Promise.all([dbs.map(db => db.close())])
+  await new Promise(r => setTimeout(r, 20000))
```

Comment:
what happens if we don't sleep here?

Reply:
This lets the nodes finish up any tasks that are currently running. Without this we get a bunch of `Connection not open on send` issues when the tests are ending.

[Without the fix](https://circleci.com/api/v1.1/project/github/zkopru-network/zkopru/2245/output/103/0?file=true&allocation-id=611374d71fb0f50ea20a6e0b-0-build%2F7E19B980)
[With the fix](https://circleci.com/api/v1.1/project/github/zkopru-network/zkopru/2275/output/103/0?file=true&allocation-id=6113807524e31916b181005c-0-build%2F1C15E1C9)
(look at the end of the logs)